browser/resources/settings/brave_overrides/personalization_options.ts

```diff
@@ -21,5 +21,18 @@ RegisterPolymerTemplateModifications({
           </settings-brave-personalization-options>
         `)
     }
+
+    // searchSugestToggle is moved to search engines section.
+    const searchSuggestToggleTemplate = templateContent.querySelector('template[is="dom-if"][if="[[showSearchSuggestToggle_()]]"]')
```

Comment:
Out of curiosity, could you just set the if condition to `false`, instead of making the content hidden?

Reply:
N/A